receiver/tests/test_routes_retention.py

```
@@ -81,14 +81,18 @@ def client(monkeypatch):
 
 
 class TestRetentionCleanupStart:
+    """Tests for POST /api/config/retention/cleanup."""
+
     def test_start_cleanup_returns_running(self, client):
-        test_client, mock_deps, mock_db, setup_mod = client
+        """Starting a cleanup returns 200 with status='running'."""
+        test_client, mock_deps, mock_db, _setup_mod = client
```

Ruff (0.15.10)

[warning] Unpacked variable `mock_db` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)